### ISSUE TITLE
Unet error in DDPM

### ISSUE THREAD

**Issue by TwinkleStarst** [labmlai/annotated_deep_learning_paper_implementations#270]
When training with the celeba dataset, the program encountered an error.

**Comment by Paschalis-D** [comment]
Yes, I am getting the same one. Did you find any solution? I'll let you know if I do.

**Comment by baizhu0414** [comment]
same issue.Have you solved the problem?
When I use dataset MNIST,it works well.

**Comment by NisargUpadhyayIITJ** [comment]
I'd like to work on this issue. Can I take it?
